src/test/java/com/google/devtools/build/docgen/generated_docs_build_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# A smoke file test for generated doc files.
+
+set -euo pipefail
+
+# --- begin runfiles.bash initialization v2 ---
+# Copy-pasted from the Bazel Bash runfiles library v2.
+set -uo pipefail; f=bazel_tools/tools/bash/runfiles/runfiles.bash
```

Comment:
This is obviously unneeded, but it also makes sense to copy over a tricky preamble like this without changes. So I'm calling it out as OK, in spite of it looking funny.